src/cross.js

```diff
@@ -334,7 +334,7 @@ export const accountDataToAccount = ({
     index,
     freshAddress,
     freshAddressPath,
-    swapHistory: swapHistory.map(fromSwapOperationRaw),
+    swapHistory: (swapHistory || []).map(fromSwapOperationRaw),
```

Comment:
This part makes no sense to me. Why would it be falsy? To me it means the error is somewhere else where we miss to set it as clearly we have defined our type to not be falsy

Reply:
Indeed

Reply:
Older LLD exports without swap types, newer LLM imports with swap types and crashes trying to iterate undefined